packages/arb-token-bridge-ui/tests/e2e/specs/withdrawERC20.cy.ts

```diff
@@ -109,6 +124,68 @@ describe('Withdraw ERC20 Token', () => {
       })
     })
 
+    it('should claim funds', { defaultCommandTimeout: 200_000 }, () => {
+      // increase the timeout for this test as claim button can take ~(20 blocks *10 blocks/sec) to activate
+
+      cy.login({ networkType: 'parentChain' }) // login to L1 to claim the funds (otherwise would need to change network after clicking on claim)
+
+      cy.findByLabelText('Open Transaction History')
+        .should('be.visible')
+        .click()
+
+      cy.waitUntil(
+        () =>
+          cy
+            .findByLabelText(
+              `Claim ${formatAmount(ERC20AmountToSend, {
+                symbol: 'WETH'
+              })}`
+            )
+            .should('be.visible')
+            .should('not.be.disabled'),
+        {
+          errorMsg: 'Claim Transaction button is not visible or enabled',
+          timeout: 200_000,
+          interval: 1000
+        }
+      ).then(() => {
+        cy.findByLabelText(
+          `Claim ${formatAmount(ERC20AmountToSend, {
+            symbol: 'WETH'
+          })}`
+        )
+          .click()
+          .then(() => {
+            cy.confirmMetamaskTransaction().then(() => {
+              cy.findByLabelText('show settled transactions')
+                .should('be.visible')
+                .click()
+
+              cy.findAllByText(
```

Comment:
Just checking, wouldn't that give false positive with different transactions, such as deposits or withdrawals tests?

Reply:
Yeah this crept in due to an old approach. Fixed.